pubsub/subscriber.go

```diff
@@ -74,14 +77,14 @@ func (s *subscriber) manager(ctx context.Context, event queuesgo.Event) bool {
 			ack, err := element.handlerFunc(ctx, event)
 			// The acknowledgment of the message is handled by the handlerFunction regardless of the error
 			if err != nil {
-				log.Printf("An error: %s for event: %s", err.Error(), eventName)
+				s.logger(fmt.Sprintf("An error: %s for event: %s", err.Error(), eventName))
 				return ack
 			}
-			log.Printf("Operation: %s was called for event", eventName)
+			s.logger(fmt.Sprintf("Operation: %s was called for event", eventName))
 			return ack
 		}
 	}
-	log.Printf("No function was registered for the event: %s", eventName)
+	s.logger(fmt.Sprintf("No function was registered for the event: %s", eventName))
```

Comment:
I feel this one shouldn't be optional :) - only the success-flow related ones (lines 60 and 83) should use `s.logger`.

What do you think?

Reply:
Agree! My mistake